### DISCUSSION TITLE
Add "companion" support for Minimed Mobile app

### DISCUSSION THREAD

**Discussion by steve8x8** [NightscoutFoundation/xDrip#2530] [Ideas]
I'm no Medtronic user, but nevertheless I'd like to see "Minimed Mobile" added to the list of supported Companion apps - the ID string of the app I've been offered by the PlayStore is "com.medtronic.diabetes.minimedmobile.eu".
In my interpretation this is the sibling to "Carelink" which connects the phone directly with the sensor, while "Minimed Mobile" links to the pump which has a BT connection with the sensor.

I have uploaded a screenshot to Gitter: 
![here](https://files.gitter.im/5e791aded73408ce4fddbfd0/zklo/photo_2022-12-03-16.01.18.jpeg)

A one-line addition has to be made to app/src/main/java/com/eveningoutpost/dexdrip/Services/UiBasedCollector.java to get the ball rolling, not sure whether that would already be sufficient?

**Comment by Navid200** [comment]
That's a record.  I have never seen a suggestion implemented so fast.  
Please check the latest Nightly just released.

**Reply by steve8x8** [reply]
Indeed, I'm impressed. Will test asap! (New version is on phone, but pump is miles away.)

**Reply by steve8x8** [reply]
... and "it just works" - as long as the phone paired with the pump is close
![photo_2022-12-07 17 07 00](https://user-images.githubusercontent.com/11962538/206232553-0f75b5e8-ad3c-4fe1-af05-d5d9a97ddad8.jpeg)

(don't judge me by the value - it's not mine, and such thing happen if the pump has to be kept in one's clothes all day, only pulled out for a check or a bolus)

(and the straight line is real, I checked with Minimed Mobile's graph, within rounding-to-one-digit errors)
